src/Integration/NewConnectedMode/IConfigurationProvider.cs

```diff
@@ -23,5 +23,7 @@ namespace SonarLint.VisualStudio.Integration.NewConnectedMode
     internal interface IConfigurationProvider : ILocalService
     {
         BindingConfiguration GetConfiguration();
+
+        bool WriteConfiguration(BindingConfiguration configuration);
```

Comment:
I went for the simplest approach of adding a _Write_ method to the _ConfigurationProvider_ (_serializer_ might now be a better name, but there is already an _ISolutionBindingSerializer_). I might refactor this later.